django/db/backends/sqlite3/operations.py

```diff
@@ -8,6 +8,8 @@
 from django.core.exceptions import FieldError
 from django.db import DatabaseError, NotSupportedError, models
 from django.db.backends.base.operations import BaseDatabaseOperations
+from django.db.backends.sqlite3.features import DatabaseFeatures
```

Comment:
You can access database features in operation methods using `self.connection.features`.

```suggestion
```